lib/telegram/bot/types/keyboard_button.rb

```diff
@@ -5,7 +5,7 @@ module Bot
     module Types
       class KeyboardButton < Base
         attribute :text, Types::String
-        attribute? :request_user, KeyboardButtonRequestUser
+        attribute? :request_users, KeyboardButtonRequestUsers
```

Comment:
Should we really replace (remove the old one)?

> The old name will still work for backward compatibility.

Reply:
TBH, I don't know if we should. I plan to release it with 2.0 version, so breaking changes are possible. What do you think?

Reply:
I see nothing what would break this library. I think the deletion of the old name will be in a new API release. But… it's just for backward compatibility, and we have our own, so it's up to you.

I personally see `request_user` with `KeyboardButtonRequestUsers` type kind of confusing.